extensions/yubikey/yubikey.extension.php

```diff
@@ -140,8 +140,11 @@ function yubikey_user_verify_credentials($uid, $credentials) {
 	if ($parts === false) {
 		log_debug('authentication for user ' . $test_user['uid'] . ' failed because the OTP doesn\'t look like one.');
 		return false;
-	}
-	if ($yubi_user['key_id'] !== $parts['prefix']) {
+	}	
+	if(!is_array($yubi_user['key_id'])) {
```

Comment:
I would prefer if you kept the same formatting rules used in the code already, i.e. a space after `if`.